tox.ini

```diff
@@ -18,23 +18,23 @@ setenv =
 passenv = *
 
 commands =
-    /usr/bin/find "{toxinidir}" -name '*.pyc' -delete
+;    /usr/bin/find "{toxinidir}" -name '*.pyc' -delete
     coverage run --include=s3_storage_provider.py -m twisted.trial test_s3
     coverage report -m
 
-[testenv:py35]
+[testenv:py37]
 usedevelop=true
 
-[testenv:py36]
+[testenv:py38]
 usedevelop=true
 
-[testenv:py37]
+[testenv:py39]
 usedevelop=true
 
-[testenv:py38]
+[testenv:py310]
 usedevelop=true
 
-[testenv:py39]
+[testenv:py311]
 usedevelop=true
```

Comment:
I think we can just do `usedevelop=true` under `[testenv]` instead of specifying it for each Python version?

Reply:
Oh, I guess we don't want to do it for `pep8` checking? Meh.

Reply:
I'm not sure what `usedevelop` actually does, tbh; we could probably rip it out? I was just trying to get a quick fix in without thinking too hard.